blockly-ws-search/src/WorkspaceSearch.js

```diff
@@ -141,18 +148,55 @@ export class WorkspaceSearch {
     const inputWrapper = document.createElement('div');
     Blockly.utils.dom.addClass(inputWrapper, 'ws-search-input');
     this.textInput_ = this.createTextInput_();
```

Comment:
I would like for us to change from restricting to textinput to just input (rename textInput to inputElement) so that we are more flexible. I don't believe it would break anything.